.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+Tips for a good bugreport
+=========================
+
+* Remember, if you are pedantically detailed about your problem, we can help you
+  better.
```

Comment:
I'd rather say something like:

* Help us help you! The more details you'll give, the better we'll be able to help you out.

being *pedantically* detailed might be understood negatively.